TheOtherRoles/Buttons.cs

```diff
@@ -239,7 +239,7 @@ public static void Postfix(HudManager __instance)
                             sabotageActive = true;
                     return sabotageActive && Engineer.remainingFixes > 0 && CachedPlayer.LocalPlayer.PlayerControl.CanMove;
                 },
-                () => {},
+                () => { if (Engineer.resetFixesAfterMeeting) Engineer.resetEngineerFixes(); },
```

Comment:
```suggestion
                () => { if (Engineer.resetFixesAfterMeeting) {
                        Engineer.resetEngineerFixes();
                    }
                 },
```

This is a stylistic nit. I recommend always having the conditional code that follows an "if (x)" type of expression within curly braces, because otherwise you can end up in strange situations like this:

if (x) y++; z++;

And in this case, y++ is conditional, but z++ always runs, because only the first statement is captured by the if. This is best avoided by just containing everything after an 'if' in curly braces to make it clear which parts are conditional.